godtools/XMLContent/KGP/kgp-5.xml

```diff
@@ -100,7 +99,7 @@
                 </content:text>
             </paragraph>
             <paragraph>
-                <content:button type="event" value="kgp-6">
+                <content:button type="event" value="kgp-6,send-information-modal">
```

Comment:
to me this would retrigger the modal you are trying to close, I don't think you want the "send-information-modal" event triggered here.